### ISSUE TITLE
blobsaver not detecting APNONCE on IOS 15 BETA 4 on IPHONE 11.

### ISSUE THREAD

**Issue by SaintsNati0n** [airsquared/blobsaver#316]
Blob saver told me to put this on GITHUB so here i am lol.

-------------------------------------------------------------

**Describe the bug**
just does not tell me the app nonce also unjailbroken if that helps.

**Screenshots**

**Desktop:**
 - OS Version: Windows 10
 - iOS Device: Iphone 11

**Checklist (place an x between the brackets to mark as completed):**
 - [ ] I have made sure I am on the latest version of blobsaver. X
 - [ ] I have checked the blobsaver wiki to see if there is already a solution to my problem. X
 - [ ] I have tried common troubleshooting tactics such as restarting my computer and reinstalling blobsaver. X

**Log(s)**
```
If applicable, paste any log(s) here, on the lines between the backticks.
```

**Comment by teelekkung** [comment]
same here but on iPhone se2020
log from terminal 

path = /Applications/blobsaver.app/Contents/Frameworks
java.lang.NullPointerException: Cannot invoke "com.sun.jna.Pointer.getByteArray(long, int)" because the return value of "airsquared.blobsaver.app.natives.Libplist.getDataPtr(com.sun.jna.Pointer, com.sun.jna.ptr.IntByReference)" is null
	at airsquared.blobsaver@3.0.1/airsquared.blobsaver.app.LibimobiledeviceUtil.plistDataToString(Unknown Source)
	at airsquared.blobsaver@3.0.1/airsquared.blobsaver.app.LibimobiledeviceUtil.getApNonceNormalMode(Unknown Source)
	at airsquared.blobsaver@3.0.1/airsquared.blobsaver.app.LibimobiledeviceUtil$GetApnonceTask.call(Unknown Source)
	at airsquared.blobsaver@3.0.1/airsquared.blobsaver.app.LibimobiledeviceUtil$GetApnonceTask.call(Unknown Source)
	at javafx.graphics/javafx.concurrent.Task$TaskCallable.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)

**Comment by yeounhojun0607** [comment]
Me too, I'm iPhone 12 mini on iOS 15 db4.

logs:

java.lang.NullPointerException: Cannot invoke "com.sun.jna.Pointer.getByteArray(long, int)" because the return value of "airsquared.blobsaver.app.natives.Libplist.getDataPtr(com.sun.jna.Pointer, com.sun.jna.ptr.IntByReference)" is null
	at airsquared.blobsaver@3.0.1/airsquared.blobsaver.app.LibimobiledeviceUtil.plistDataToString(Unknown Source)
	at airsquared.blobsaver@3.0.1/airsquared.blobsaver.app.LibimobiledeviceUtil.getApNonceNormalMode(Unknown Source)
	at airsquared.blobsaver@3.0.1/airsquared.blobsaver.app.LibimobiledeviceUtil$GetApnonceTask.call(Unknown Source)
	at airsquared.blobsaver@3.0.1/airsquared.blobsaver.app.LibimobiledeviceUtil$GetApnonceTask.call(Unknown Source)
	at javafx.graphics/javafx.concurrent.Task$TaskCallable.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)

**Comment by teelekkung** [comment]
.

**Comment by airsquared** [comment]
This has been fixed in v3.0.2.
